test/Riok.Mapperly.Tests/GeneratedMethod.cs

```diff
@@ -8,15 +9,26 @@ public GeneratedMethod(MethodDeclarationSyntax declarationSyntax)
     {
         Name = declarationSyntax.Identifier.ToString();
         Signature = $"{declarationSyntax.ReturnType.ToString()} {Name}{declarationSyntax.ParameterList.ToString().Trim()}";
+        ConstraintClauses = ExtractParameterConstraints(declarationSyntax.ConstraintClauses);
         Body = ExtractBody(declarationSyntax);
     }
 
     public string Name { get; }
 
     public string Signature { get; }
 
+    public string? ConstraintClauses { get; }
```

Comment:
Couldn't we simply add this to the signature? IMO this is part of the signature in C#...

Reply:
We can (I was searching for it there originally). I personally like that it's also a separate property (for these constraints-specific checks).

Is there a problem if it is duplicated i.e.:
```cs
ConstraintClauses = ExtractParameterConstraints(declarationSyntax.ConstraintClauses);
Signature = $"{declarationSyntax.ReturnType.ToString()} {Name}{declarationSyntax.ParameterList.ToString().Trim()}{ConstraintClauses}";
```

Reply:
Works with me 👍